extensions/ql-vscode/src/queries-panel/query-discovery.ts

```diff
@@ -36,16 +30,15 @@ interface QueryDiscoveryResults {
 export class QueryDiscovery extends Discovery<QueryDiscoveryResults> {
   private results: QueryDiscoveryResults | undefined;
 
-  private readonly onDidChangeQueriesEmitter = this.push(
-    new EventEmitter<void>(),
-  );
+  private readonly onDidChangeQueriesEmitter;
```

Comment:
Can we add a type here? I think this is now typed as `any`.

Reply:
Ah thanks for pointing this out. For me locally it was working out the correct type. Perhaps it's smart enough to see the single assignment in the constructor.

But I do kinda like things to be explicit and readable to the human eye, so putting a type here sounds good to me regardless.